src/main/java/org/jahia/modules/contenteditor/api/forms/EditorFormField.java

```diff
@@ -58,32 +62,37 @@ public EditorFormField(String name,
 
     public EditorFormField(EditorFormField field) {
         this(
-                field.name,
-                field.selectorType,
-                field.selectorOptions == null ? null : field.selectorOptions.stream()
-                        .map(option -> new EditorFormProperty(option))
-                        .collect(Collectors.toList()),
-                field.i18n,
-                field.readOnly,
-                field.multiple,
-                field.mandatory,
-                field.valueConstraints == null ? null : field.valueConstraints.stream()
-                        .map(constraint -> new EditorFormFieldValueConstraint(constraint))
-                        .collect(Collectors.toList()),
-                field.defaultValues == null ? null : new ArrayList<>(field.defaultValues),
-                field.removed,
-                field.targets == null ? null : field.targets.stream()
-                        .map(target -> new EditorFormFieldTarget(target))
-                        .collect(Collectors.toList()),
-                // FIXME no deep copy here
-                field.extendedPropertyDefinition
+            field.name,
+            field.nodeType,
+            field.selectorType,
+            field.selectorOptions == null ? null : field.selectorOptions.stream()
+                .map(option -> new EditorFormProperty(option))
+                .collect(Collectors.toList()),
+            field.i18n,
+            field.readOnly,
+            field.multiple,
+            field.mandatory,
+            field.valueConstraints == null ? null : field.valueConstraints.stream()
+                .map(constraint -> new EditorFormFieldValueConstraint(constraint))
+                .collect(Collectors.toList()),
+            field.defaultValues == null ? null : new ArrayList<>(field.defaultValues),
+            field.removed,
+            field.targets == null ? null : field.targets.stream()
+                .map(target -> new EditorFormFieldTarget(target))
+                .collect(Collectors.toList()),
+            // FIXME no deep copy here
```

Comment:
Why not Fix this in the PR ?

Reply:
Not related to this task. We should create dedicated task to fix this one.

Reply:
The task is created ?

If not, I think you should create it :)

Reply:
Ok, I'll created.

Reply:
After Discussion with @cedricmoitrier:
We don't see the usage of keeping `extendedPropertyDefinition`.

We would like to hear from you @sergehuber